fsutil/fsutil_windows.go

```diff
@@ -0,0 +1,70 @@
+// +build windows
+
+package fsutil
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+)
+
+// CleanPath resolves common aliases in a path and cleans it as much as possible
+func CleanPath(path string) string {
+	// http://stackoverflow.com/questions/17609732/expand-tilde-to-home-directory
+	if path[:2] == "~/" {
+		usr, _ := user.Current()
+		dir := usr.HomeDir
+		path = strings.Replace(path, "~/", dir+"/", 1)
+	}
+	if p, err := filepath.Abs(path); err == nil {
+		return p
+	}
+	return filepath.Clean(path)
+}
+
+// IsDir checks if a certain path exists and is a directory
+// https://stackoverflow.com/questions/10510691/how-to-check-whether-a-file-or-directory-denoted-by-a-path-exists-in-golang
+func IsDir(path string) bool {
+	fi, err := os.Lstat(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			// not found
+			return false
+		}
+		fmt.Printf("failed to check dir %s: %s\n", path, err)
+		return false
+	}
+	if fi.Mode()&os.ModeSymlink != 0 {
+		fmt.Printf("dir %s is a symlink. ignoring", path)
+		return false
+	}
+
+	return fi.IsDir()
+}
+
+// IsFile checks if a certain path is actually a file
+func IsFile(path string) bool {
+	fi, err := os.Lstat(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			// not found
+			return false
+		}
+		fmt.Printf("failed to check dir %s: %s\n", path, err)
+		return false
+	}
+	if fi.Mode()&os.ModeSymlink != 0 {
+		fmt.Printf("dir %s is a symlink. ignoring", path)
+		return false
+	}
+
+	return fi.Mode().IsRegular()
+}
+
+// Tempdir returns a temporary directory suiteable for sensitive data. On
+// Windows, just return empty string for ioutil.TempFile.
+func Tempdir() string {
```

Comment:
The Tempdir function seems to be the only difference between fsutil_windows.go and fsutil_others.go.

Could you leave the common code in fsutil.go and only put the Tempdir function in fsutil_windows.go / fsutil_others.go?